### PR TITLE
Implemented DOMElement::getElementById()

### LINKED ISSUE #648: DOMDocument->getElementById() NotImplementedException
### Description
The `DOMElement::getElementById()` method seems not to be implemented:

https://github.com/peachpiecompiler/peachpie/blob/55ab289c1f438237c93825c5d6e16ca2a3ad5370/src/Peachpie.Library.XmlDom/DOMDocument.cs#L483-L489

Could you please implement it?

### PR DESCRIPTION
**Pull request by DRSDavidSoft**
### Scope
- fixes #648

### CODE REVIEW

**Comment by DRSDavidSoft** [comment]
It's funny, I don't see why this hasn't been not implemented before 😄

File: src/Peachpie.Library.XmlDom/DOMDocument.cs [hunk 1]
```diff
@@ -481,11 +481,14 @@ public virtual DOMNodeList getElementsByTagNameNS(string namespaceUri, string lo
...
-        /// Not yet implemented.
+        /// Gets the first element with the matching ID attribute.
         /// </summary>
+        /// <param name="elementId">The attribute ID to match.</param>
+        /// <returns>A <see cref="DOMElement"/>.</returns>
         public virtual DOMElement getElementById(string elementId)
         {
-            throw new NotImplementedException();
+            XmlElement element = XmlDocument.GetElementById(elementId);
+            return new DOMElement(element);
```
Comment (jakubmisek) [inline src/Peachpie.Library.XmlDom/DOMDocument.cs]: the function `getElementById` returns `null` if the element is not found

**Comment by jakubmisek** [comment]
Thanks! the [documentation](http://docs.php.net/manual/en/domdocument.getelementbyid.php) is little bit more complex, but I think .NET handles it well :)

File: src/Peachpie.Library.XmlDom/DOMDocument.cs (hunk 1 above)
Comment (DRSDavidSoft) [reply src/Peachpie.Library.XmlDom/DOMDocument.cs]: @jakubmisek So, `DOMElement(null)` does not return `null`?

Comment (jakubmisek) [reply src/Peachpie.Library.XmlDom/DOMDocument.cs]: `new DOMElement` certainly does not :)

Comment (DRSDavidSoft) [reply src/Peachpie.Library.XmlDom/DOMDocument.cs]: @jakubmisek Which one would you suggest is the better style?

### A
```cs
return element == null ? element : new DOMElement(element);
```

### B
```cs
return element == null ? null : new DOMElement(element);
```

### C
```cs
return element != null ? new DOMElement(element) : null;
```

Comment (jakubmisek) [reply src/Peachpie.Library.XmlDom/DOMDocument.cs]: haha :) `C` is fine - the most frequent branch first

Comment (DRSDavidSoft) [reply src/Peachpie.Library.XmlDom/DOMDocument.cs]: @jakubmisek Awesome, never thought of it that way.
I just committed this for review

**Comment by jakubmisek** [comment]
thanks :)

**Pull request merged**